lib/profiling/lfric/timer_mod.F90

```diff
@@ -0,0 +1,302 @@
+!-----------------------------------------------------------------------------
+! (C) Crown copyright 2017 Met Office. All rights reserved.
+! For further details please refer to the file LICENCE which you should have
```

Comment:
Please correct and add BSD for modifications.

Reply:
This file is gone.